packages/components/src/components/hds/app-side-nav/list/index.hbs

```diff
@@ -0,0 +1,19 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<nav class="hds-app-side-nav__list-wrapper" ...attributes>
+  {{yield (hash ExtraBefore=(component "hds/yield"))}}
+  <ul class="hds-app-side-nav__list" role="list">
```

Comment:
Is there a way for the title to be added to the list as the value for an `aria-labelledby` attribute? 

Otherwise it will continue to show up in rotors as "list" instead of having a meaningful name. It will also be indistinguishable from other lists on the page.

Reply:
https://github.com/user-attachments/assets/cd34a7a5-855d-4482-b0dd-684c92fbf098

Here's a video to show what I mean.

Reply:
I added it to the list. I would like to make updates like this as a follow on if that's ok.

Reply:
I looked at this and I don't see a way to really do it with the current structure. I believe it would requiring completely restructuring the code for the "Title" in relation to the List and List Items.

I worked on the SideNav in the very early beginning stages and had structured the title to be outside the list instead of inside it but was overruled. The structure and functionality of the current SideNav (and now the AppSideNav which is based off it) is very complex so I'm wary of making too many changes to the structure if it can be avoided.